charts/tsa/README.md

```diff
@@ -95,74 +95,3 @@ To enabled access from external resources, an Ingress resource is created. The c
 >         - host: timestamp.localhost
 >           path: /
 > ```
-
-## Quick Installation
```

Comment:
This doesn't look to be right!

Reply:
Ah, this is because I need to move this to `README.md.gotmpl`.